graphbolt/src/csc_sampling_graph.cc

```diff
@@ -115,5 +117,35 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::InSubgraph(
           : torch::nullopt);
 }
 
+c10::intrusive_ptr<CSCSamplingGraph>
+CSCSamplingGraph::BuildGraphFromSharedMemoryTensors(
+    std::tuple<
```

Comment:
why not use `SharedMemoryTensors` defined in `shared_memory_utils.h`? for clarity?

Reply:
Yes, I think it is more clear for a header file. And also `shared_memory_utils.h` is not in the include directory and I don't want to move it.